src/site/stages/build/process-cms-exports/transformers/paragraph-lists_of_links.js

```diff
@@ -5,11 +5,13 @@ const { getDrupalValue } = require('./helpers');
 const transform = entity => ({
   entityType: 'paragraph',
   entityBundle: 'lists_of_links',
-  fieldSectionHeader: entity.fieldSectionHeader,
-  fieldVaParagraphs: entity.fieldVaParagraphs,
+  entity: {
+    fieldSectionHeader: getDrupalValue(entity.fieldSectionHeader),
+    fieldVaParagraphs: entity.fieldVaParagraphs,
+  },
 });
 
 module.exports = {
-  filter: ['fieldSectionHeader', 'fieldVaParagraphs'],
+  filter: [],
```

Comment:
it should be `filter: ['field_section_header', 'field_va_paragraphs'],`

Reply:
Do we really need a filter here? I don't think we do and it makes it more confusing to debug

Reply:
Added it for consistency, but we should explore removing it in all transformers in the future, I really don't see how it's useful 🙂